### PR TITLE
[MIG] project_task_send_by_mail: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

File: project_task_send_by_mail/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Task Send By Mail",
+    "summary": "Send task report by email",
+    "version": "11.0.1.0.0",
+    "author": "Tecnativa, "
+               "Odoo Community Association (OCA)",
```
Comment (chienandalu) [inline project_task_send_by_mail/__manifest__.py]: the indenting here isn't correct

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tested on runbot. Just review LINT and a minor comment below.

File: project_task_send_by_mail/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+=========================
```
Comment (pedrobaeza) [inline project_task_send_by_mail/readme/DESCRIPTION.rst]: Remove the title here

**Comment by cubells** [comment]
All done

**Review by pedrobaeza — APPROVED** [review]

**Review by tafaRU — APPROVED** [review]
Code review and test.

**Pull request merged**